core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -17,6 +17,33 @@
         background: #eeeeee no-repeat center center fixed;
         background-size: cover;
       }
+      .exploration-list-table {
```

Comment:
Prefix this and the following classes with 'oppia-', to distinguish them from classes that come in via bootstrap (say).